src/core_landice/mode_forward/mpas_li_iceshelf_melt.F

```diff
@@ -1166,6 +1166,7 @@ end subroutine iceshelf_melt_ismip6
 
     subroutine grounded_face_melt_ismip6(domain, err)
```

Comment:
Variables no longer used in this routine that should be deleted:
```
scratchPool
nEdgesSolve
maxEdges
calvingVelocity
areaCell is used but is retrieved twice
cellsOnEdge
lowerSurface
groundedMarineMarginMask
faceMeltRateApplied & deltat are currently used but only for the now retired distribute_face_melt_flux routine, so should be deleted.  faceMeltRateApplied will need to be deleted from Registry as well.
oceanEdgeLength & submergedFaceArea also appear to be vestigial calculations from the old distribute... routine
```

Reply:
These have been removed, except for lowerSurface, which is used to calculate faceMeltSpeedVertAvg. dvEdge also removed.